packages/in_app_purchase/in_app_purchase_storekit/example/lib/main.dart

```diff
@@ -12,12 +12,11 @@ import 'consumable_store.dart';
 import 'example_payment_queue_delegate.dart';
 
 void main() {
+  InAppPurchaseStoreKitPlatform.enableStoreKit2();
```

Comment:
This is the example app - storekit2 needs to be called before everything else to ensure no transactions get missed.